roles/quay/tasks/main.yml

```diff
@@ -0,0 +1,19 @@
+---
+
+- name: Set Authorization Header when specifying API token
+  set_fact:
+    auth_header: "Bearer {{ quay_api_token }}"
+  when: (quay_api_token|trim|length) > 0
+
+- name: Assert Authorization header has been set
+  assert:
+    that:
+      (quay_api_token|trim|length) > 0
+    fail_msg: "Quay API Token or username and password must be specified"
```

Comment:
Doesn't look like username/password is supported with how it is currently implemented, so maybe update this message to avoid confusion?